tailwind.config.js

```diff
@@ -3,8 +3,15 @@ module.exports = {
   content: ["./*.{html,js}", "./src/**/*.{html,js}"],
   theme: {
     extend: {
-      lineHeight: {
-        'extra-loose': '3.5rem',
+      height: {
+        'sm-reference': '130vh',
+        'md-reference': '120vh',
+        'lg-reference': '81.25rem'
+      },
+      width: {
+        'w-46-reference': '46vh',
+        'w-reference':'25rem'
```

Comment:
We already have width and height in tailwind classes, use them and avoid creating new tailwind classes where they are not necessary.